src/control-flow-basics/break-continue/labels.md

```diff
@@ -0,0 +1,35 @@
+# labels
+
+Both `continue` and `break` can optionally take a label argument which is used
+to break out of nested loops:
+
+```rust,editable
+fn main() {
+    let (mut a, b) = (52, 100);
+    let result = loop {
+        if a > b {
+            break a;
+        } else {
+            a += 20;
+        }
+        let mut i = 0;
+        'outer: for _x in 1..5 {
```

Comment:
This actually had me scratching my head for a few minutes -- the label says `'outer` but it's the _inner_ loop! Also, without a label `break` will break the innermost loop, so this doesn't actually show the power of labels. Can you update the example so that the outer loop is the one labeled? I think that will require breaking with a value in that case, too.